src/interactive_marker_server.cpp

```diff
@@ -388,6 +388,17 @@ bool InteractiveMarkerServer::get( std::string name, visualization_msgs::Interac
   return false;
 }
 
+void InteractiveMarkerServer::getNames(std::vector<std::string>& names)
```

Comment:
Is there a particular reason you chose to use an output parameter instead of returning a vector? e.g.

```c
std::vector<std::string> InteractiveMarkerServer::getNames()
```

IMO, returning the vector seems more intuitive.

Reply:
not a specific reason really, we can return the vector